conans/test/integration/command/create_test.py

```diff
@@ -572,6 +572,8 @@ def package_info(self):
                 self.cpp_info.set_property("pkg_config_name", "pkg_other_name")
                 self.cpp_info.set_property("pkg_config_aliases", ["pkg_alias1", "pkg_alias2"])
                 self.cpp_info.components["cmp1"].libs = ["libcmp1"]
+                self.cpp_info.components["cmp1"].libdirs = ["lib"]
```

Comment:
These tests were ill-formed in the first place, because they were defining things that doesn't work. Yes, they are serialized, but most generators just discard the global cpp_info definition.